File: Src/Services/Identity/UserConfirmation.cs
using Microsoft.AspNetCore.Identity;
using MusicPlayerBackend.Data.Entities;

namespace MusicPlayerBackend.Services.Identity;

public sealed class UserConfirmation : IUserConfirmation<User>
{
    public Task<bool> IsConfirmedAsync(UserManager<User> manager, User user)
    {
        return Task.FromResult(true);
    }
}

File: Src/MusicPlayerBackend.Identity/UserConfirmation.fs
namespace MusicPlayerBackend.Identity

open System.Threading.Tasks
open Microsoft.AspNetCore.Identity
open MusicPlayerBackend.Data.Entities

// TODO: Implement mail confirmation
[<Sealed>]
type UserConfirmation() =
    interface IUserConfirmation<User> with
        member this.IsConfirmedAsync(_, _) =
            Task.FromResult(true)
